lib/angular-evaporate.js

```diff
@@ -147,6 +157,11 @@
                       (100 - that.$percent) / that.$percent *
                       (now - that.$started) / 1000
                     );
+                    var progress = (arg * 100).toFixed(2);
+                    var delta = (new Date() - that.$started) / 1000;
+                    var loaded = (progress * that.sizeBytes) / 100;
+                    var totalSentBySecond = loaded / delta;
```

Comment:
`var` declaration inside a block scope is not a good practice because of [hoisting](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/var#var_hoisting)